backend/app/api/endpoints/openapi_responses.py

```
@@ -0,0 +1,550 @@
+    result = {
+        "namespace": parts[0],
+        "team_name": parts[1],
+        "model_id": parts[2] if len(parts) > 2 else None,
+    }
+    return result
+
+
+def _task_to_response_object(
+    task_dict: Dict[str, Any],
+    model_string: str,
+    subtasks: list = None,
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)